Source/Immutable/Public/Immutable/ImmutableDataTypes.h

```diff
@@ -6,6 +6,9 @@
 
 #include "ImmutableDataTypes.generated.h"
 
+// This is the version of the SDK that is being used. This is not the version of the engine.
+// This hardcoded value will be updated by a workflow during the release process.
+#define ENGINE_SDK_VERSION TEXT("1.2.4")
```

Comment:
`SDK version` = our ts sdk version
`Engine version` = unreal or unity's engine's version, because of the word engine. (e.g. ue4)
`Game Bridge version` = the version game bridge (currently always the same as SDK version i think)
`Game Client Release Version` = what we want here. in this case I suggest `UNREAL_SDK_RELEASE_VERSION` or `GAME_CLIENT_RELEASE_VERSION` or `GAME_SDK_RELEASE_VERSION` to avoid the word "engine".

Reply:
Happy to use GAME_SDK_RELEASE_VERSION here. I chose `ENGINE_SDK_VERSION` here because that maps to the same field in the data dashboard, so its easy to trace the flow of information and its consistent with the Unity SDK naming too

Reply:
> same field in the data dashboard
who (what) named the field this way in the "data dashboard"?